bin/sort-fields

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/gawk -f
+# vim:ts=2:sw=2:noet
+
+function tmpfile_name(filename) {
+  return "." filename ".tmp"
+}
+
+function panic(msg) {
+	print FILENAME ":" FNR ": " msg > "/dev/stderr"
+	exit(1)
+}
+
```

Comment:
Does the extra newline have significance here?

Reply:
Nah all this whitespace is just the result of me typing faster than I'm reading. I'll clean it all up.